### PR TITLE
DM-40413: Update with USDF information

### PR DESCRIPTION
**Pull request by bsmartradio**
Update information and links to reflect the change to USDF

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1012,10 +900,11 @@ The schema registry therefore gets a DNS A record 'alert-schemas-int.lsst.cloud'
 
 Configuring a new Phalanx deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The following information is only rlevant if you are setting up an independant alert broker environment.
```
Comment (ebellm) [inline index.rst]:
```suggestion
The following information is only relevant if you are setting up an independent alert broker environment.
```

File: index.rst [hunk 2]
```diff
@@ -1174,7 +1064,7 @@ This would need to be replaced with something appropriate for the targeted envir
 The requirements are made clear in the ``storage.py`` files of the `github.com/lsst-dm/alert_database_ingester`_ and `github.com/lsst-dm/alert_database_server`_ repositories.
 An implementation would need to fulfill the abstract interface provided in that file.
 
-There may be more requirements, but these are certainly to need investigation if you're planning to move to a different Kubernetes provider.
+There may be more requirements, but there certianly needs to be  an investigation if you're planning to move to a different Kubernetes provider.
```
Comment (ebellm) [inline index.rst]:
```suggestion
There may be more requirements, but there certainly needs to be an investigation if you're planning to move to a different Kubernetes provider.
```

**Review by ebellm — APPROVED** [review]
Couple of small typos--I haven't walked through the whole guide as a user as I don't have ArgoCD access.

File: index.rst (hunk 1 above)
Comment (bsmartradio) [reply index.rst]: Fixed

File: index.rst (hunk 2 above)
Comment (bsmartradio) [reply index.rst]: Fixed

**Pull request merged**